pkg/dockerfile/fast_generator.go

```diff
@@ -16,10 +17,10 @@ import (
 )
 
 const FUSE_RPC_WEIGHTS_PATH = "/srv/r8/fuse-rpc/weights/sha256"
-const MONOBASE_CACHE_PATH = "/var/cache/monobase"
-const APT_CACHE_MOUNT = "--mount=type=cache,target=/var/cache/apt,id=apt-cache,sharing=locked"
```

Comment:
We're not caching `/var/cache/apt` for the APT tarball building stage. But those are usually small and rarely change. We can add cache later if it becomes problematic.